parameters/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+//! # Parameters
+//! Offer a centra place to store and configure parameters.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::unused_unit)]
+
+use frame_support::pallet_prelude::*;
+use frame_system::pallet_prelude::*;
+
+use frame_support::traits::EnsureOriginWithArg;
+use orml_traits::parameters::{AggregratedKeyValue, Key, ParameterStore};
+
+mod mock;
+mod tests;
+mod weights;
+
+pub use module::*;
+pub use weights::WeightInfo;
+
+#[frame_support::pallet]
+pub mod module {
+	use super::*;
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+
+		/// The key value type for parameters. Usually created by
+		/// orml_traits::parameters::define_aggregrated_parameters
+		type AggregratedKeyValue: AggregratedKeyValue;
```

Comment:
If I understand well, "AggregratedKeyValue" is an extensive way of saying "Entry"?